File: FinanceApp.Shared/Dto/CreditCard/CreateCreditCard.cs
using FinanceApp.Shared.Dto.Base;

namespace FinanceApp.Shared.Dto.CreditCard
{
    public class CreateCreditCard : CreateDto
    {
        public string Name { get; set; }
        public int InvoiceClosingDay { get; set; }
        public int InvoicePaymentDay { get; set; }

    }
}

File: FlutterApp/finance_app/lib/models/credit_card/create_credit_card.dart
class CreateCreditCard {
  String name;
  int invoiceClosingDay;
  int invoicePaymentDay;

  CreateCreditCard(
      {required this.name,
      required this.invoiceClosingDay,
      required this.invoicePaymentDay});

  CreateCreditCard.fromJson(Map<String, dynamic> json)
      : name = json['name'],
        invoiceClosingDay = json['invoiceClosingDay'],
        invoicePaymentDay = json['invoicePaymentDay'];

  Map<String, dynamic> toJson() => {
        'name': name,
        'invoiceClosingDay': invoiceClosingDay,
        'invoicePaymentDay': invoicePaymentDay,
      };
}
